docs/src/_static/custom.css

```diff
@@ -0,0 +1,21 @@
+html[data-theme="dark"], html[data-theme="light"] {
+    --pst-color-primary: #3399ff;
+    --pst-color-secondary: #9580ff;
+}
+
+a {
+    text-decoration: none;
+}
+
+a:hover {
+    text-decoration: underline;
+}
+
```

Comment:
It's a little awkward to show two underlines for the navigation bar links.
![image](https://github.com/y-scope/clp/assets/43196707/e8caaf1f-4120-4640-a60c-ddc480e2de51)

Maybe we should remove the underline for all links in the header.

```suggestion
a:hover {
    text-decoration: underline;
}

.bd-header a:hover {
    text-decoration: none !important;
}
```